src/components/atm.typography/typography.styled.jsx

```diff
@@ -1,9 +1,24 @@
 import styled from 'styled-components'
-import {FontSize, Color, FontFamily, FontWeight} from '../constants'
+import {FontSize, Color, FontFamily, FontWeight, Spacing} from '../constants'
+
+const FontColor = {
+    black: Color.Black,
+    white: Color.White
+
+}
 
 export const H1 = styled.h1`
     font-size: ${FontSize.Large};
-    color: ${Color.White};
+    color: ${props => FontColor[props.color] ?? Color.Black};
```

Comment:
vc sabe se queremos ou chegaremos a ter outras cores aqui alem de branco e preto? caso contrario, podia ser um boolean props como 'light', trocando preto por branco só. Pra fechar porta e n introduzir mais cores no futuro

Reply:
São apenas as duas variações. Realmente faz mais sentido quando for dois ele ser uma props direto e não um objeto de cores.